data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -98,7 +98,8 @@ then
 		--manifest-etcd-server-urls={{.EtcdCluster}} \
 		--manifest-image=${OPENSHIFT_HYPERSHIFT_IMAGE} \
 		--asset-input-dir=/assets/tls \
-		--asset-output-dir=/assets/kube-apiserver-bootstrap \
+		{{ if ne .CertRotationBase "" }}--unsupported-cert-rotation-base={{.CertRotationBase}} \
+{{ end }}		--asset-output-dir=/assets/kube-apiserver-bootstrap \
```

Comment:
so pretty

Reply:
Would the following be more pleasant?

```
    {{ if ne .CertRotationBase "" -}}
    --unsupported-cert-rotation-base={{.CertRotationBase}} \
    {{ end -}}
    --asset-output-dir=/assets/kube-apiserver-bootstrap \
```

Reply:
@staebler thanks, I was hoping something like that existed but haven't got the time to dig :+1: updated